dali/test/python/test_dali_tf_dataset_graph.py

```diff
@@ -140,23 +135,23 @@ def test_tf_dataset_with_stop_iter():
         for max_shape in [(10, 20), (120, 120, 3), (3, 40, 40, 4)]:
             for dtype in [np.uint8, np.int32, np.float32]:
                 for iters in [1, 2, 3, 4, 5]:
-                    yield run_tf_dataset_with_stop_iter, dev, max_shape, dtype, iters * batch_size - 3
+                    yield run_tf_dataset_with_stop_iter, dev, \
+                          max_shape, dtype, iters * batch_size - 3
 
 
 def run_tf_dataset_multi_input(dev, start_values, input_names, batches):
-    run_tf_dataset_graph(dev,
+    run_tf_dataset_graph(
+        dev,
         get_pipeline_desc=external_source_tester_multiple(start_values, input_names, batches),
         to_dataset=external_source_converter_multiple(start_values, input_names, batches))
 
 
-start_values = [[np.full((2, 4), -42, dtype=np.int64),
-                 np.full((3, 5), -123.0, dtype=np.float32)],
-                [np.full((3, 5), -3.14, dtype=np.float32)],
-                [
-                    np.full((2, 4), -42, dtype=np.int64),
-                    np.full((3, 5), -666.0, dtype=np.float32),
-                    np.full((1, 7), 5, dtype=np.int8)
-                ]]
+start_values = [
+    [np.full((2, 4), -42, dtype=np.int64), np.full((3, 5), -123.0, dtype=np.float32)],
+    [np.full((3, 5), -3.14, dtype=np.float32)],
+    [np.full((2, 4), -42, dtype=np.int64), np.full((3, 5), -666.0, dtype=np.float32),
+     np.full((1, 7), 5, dtype=np.int8)]
+]
```

Comment:
This looks like two autoformatters fighting.... Was the previous variant somehow offending?

Reply:
The formatting was inconsistent. First two elements of `start_values` begun in the same line as the opening bracket `[`, while the last one had newline inserted between. I've made it consistent.
```python
start_values = [[np.full((2, 4), -42, dtype=np.int64),
                 np.full((3, 5), -123.0, dtype=np.float32)],
                [np.full((3, 5), -3.14, dtype=np.float32)],
                [
                    np.full((2, 4), -42, dtype=np.int64),
                    np.full((3, 5), -666.0, dtype=np.float32),
                    np.full((1, 7), 5, dtype=np.int8)
                ]]
```